src/gen-arabic-table.py

```diff
@@ -5,7 +5,7 @@
 if len (sys.argv) != 4:
 	print ("""usage: ./gen-arabic-table.py ArabicShaping.txt UnicodeData.txt Blocks.txt
 
-Input files, as of Unicode 12:
```

Comment:
better to use Unicode 13 here as an indication the script has recognized to be compatible with the indicated Unicode version I believe.

Reply:
All these URLs include “latest”: they aren’t specific to Unicode 13.